nearlib/signing/key_pair.js

```diff
@@ -1,23 +1,39 @@
-/**
- * Key pair.
- */
+
 const bs58 = require('bs58');
 const nacl = require('tweetnacl');
 
+/**
+ * This class provides key pair functionality (generating key pairs, encoding key pairs).
+ */
 class KeyPair {
+    /**
+     * Construct an instance of key pair given a public key and secret key. It's generally assumed that these
+     * are encoded in bs58.
+     * @param {string} publicKey 
+     * @param {string} secretKey 
+     */
     constructor(publicKey, secretKey) {
         this.publicKey = publicKey;
         this.secretKey = secretKey;
     }
 
+    /**
+     * Get the public key.
+     */
     getPublicKey() {
         return this.publicKey;
     }
 
+    /**
+     * Get the secret key.
+     */
     getSecretKey() {
         return this.secretKey;
     }
 
+    /**
+     * Static method to generate a new keypair from a random seed
```

Comment:
nit: Let's remove stuff like "Static method", "This class", etc. It's obvious when something is a class, or a static method.

Reply:
removed